subscription-service/src/main/kotlin/com/egm/stellio/subscription/utils/DBConversionUtils.kt

```diff
@@ -0,0 +1,23 @@
+package com.egm.stellio.subscription.utils
```

Comment:
There are the same file in `search_service`. Maybe put them in `shared` folder ?